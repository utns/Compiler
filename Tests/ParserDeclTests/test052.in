type
    point = record
        x, y: float;
    end;
var
    start: point;