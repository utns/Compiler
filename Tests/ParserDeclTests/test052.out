integer                  type 
float                    type 
char                     type 
point              type alias          record
     x                         var           float
     y                         var           float
start                     var           point
